### ISSUE TITLE
Hardset path for zpool

### ISSUE THREAD

**Issue by vhns** [zbm-dev/zfsbootmenu#207]
Hi,

It seems that in 90zfsbootmenu/module-setup.sh the path for zpool is set statically. This generates the following issue upon trying to generate an image with dracut: 

ldd: /usr/bin/zpool: No such file or directory

I believe the fix should be the following (?):
```
116c116
<   if ldd /usr/bin/zpool | grep -qF 'libgcc_s.so'; then
---
>   if ldd $(whereis -b zpool | awk '{print $2;}') | grep -qF 'libgcc_s.so'; then
```

Best regards,

vhns

**Comment by zdykstra** [comment]
What distribution are you running and where is `zpool` on your system?

**Comment by vhns** [comment]
I am running gentoo. It's in /sbin/zpool.

**Comment by zdykstra** [comment]
This has been fixed in 2d03ff5 . We're entering a feature freeze this week and will be testing/tagging/building a new release later this month.
